custom-elements/perform-microtask-checkpoint-before-construction-expected.txt

```diff
@@ -0,0 +1,2 @@
+PASS HTML parser must perform a microtask checkpoint before constructing a custom element
+PASS HTML parser must perform a microtask checkpoint before constructing a custom element during the adoption agency algorithm
```

Comment:
We shouldn't be adding `-expected` files to WPT. These are WebKit specific files meant to keep track of the current test result.

Reply:
Thanks, I have removed it now!